content/api-reference/queries/get-activities.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Get activities
+description: Retrieve all or very specific activities in your tenant
+---
+
+## Request
+
+### Query
+
+```graphql
+
+```
+
+### Variables
+
+```json
+
+```
```

Comment:
```suggestion
```json
{  
    "activity_type": [],
    "activity_status": [],
    "pathway_definition_id": [],
    "action": [],
    "patient_id": null, // optional, will not filter by patient id if null or variable omitted
    "offset": 10,
    "count": 10
}
```
```